app/settings.py

```diff
@@ -66,10 +66,20 @@ def get_env_or_fail(key):
 EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS = ensure_min(os.getenv('EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS', 10000),
                                                        1000)
 
-EQ_DYNAMODB_ENABLED = parse_mode(os.getenv('EQ_DYNAMODB_ENABLED', 'True'))
-if EQ_DYNAMODB_ENABLED:
-    EQ_DYNAMODB_ENDPOINT = os.getenv('EQ_DYNAMODB_ENDPOINT')
-    EQ_SUBMITTED_RESPONSES_TABLE_NAME = get_env_or_fail('EQ_SUBMITTED_RESPONSES_TABLE_NAME')
+EQ_DYNAMODB_ENDPOINT = os.getenv('EQ_DYNAMODB_ENDPOINT')
+EQ_DYNAMODB_MAX_RETRIES = int(os.getenv('EQ_DYNAMODB_MAX_RETRIES', 1))
```

Comment:
Im in 2 minds about the retries. Do we want to make the user wait a little bit longer if we hit DB capacity?
Is it better to eventually save the users input rather than error?

Reply:
The retry backoff is described here https://github.com/boto/botocore/blob/develop/botocore/data/_retry.json#L108.  So seconds wait between retries would be: `0.05, 0.1, 0.2, 0.4, 0.8, 1.6, 3.2, 6.4, 12.8, 25.6`.  I think we could afford a few retries, say 5?

Reply:
yeh, will give that a go.